internal/pkg/consensus/testing.go

```diff
@@ -57,18 +59,20 @@ type FakePowerTableViewSnapshot struct {
 }
 
 // Query produces test logic in response to PowerTableView queries.
-func (tq *FakePowerTableViewSnapshot) Query(ctx context.Context, optFrom, to address.Address, method string, params ...interface{}) ([][]byte, error) {
-	if method == "getTotalStorage" {
+func (tq *FakePowerTableViewSnapshot) Query(ctx context.Context, optFrom, to address.Address, method types.MethodID, params ...interface{}) ([][]byte, error) {
+	// Note: this currently happens to work as is, but it's wrong
```

Comment:
The code was working before because the method names happen to not be repeated between actors, but it was not correct nor is it correct. Name collision chances were a bit lower than ID collisions which all start at the same number between actors.